### PR TITLE
Init base properties

### PR DESCRIPTION
**Pull request by devGnode**

### CODE REVIEW

File: src/main/java/fr/zenity/desbugs/Test.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        ScenarioContext context = new ScenarioContext();
+
+        context.set(TEST,"msdrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrr");
+        context.set(LOL,new Object[]{ ScenarioContext.class });
+        System.out.println( (String)context.get(TEST) );
+        System.out.println( (Object)context.get(LOL).toString() );
+
+        System.out.println( Config.propConfig.device.isRealDevice() );*/
+
+        //LocalDriver localDriver = new LocalDriver();
```
Comment (l-bachmann) [inline src/main/java/fr/zenity/desbugs/Test.java]: A lot of commented code ^^.
I think you can deleted this class ; it was the a test.

File: src/main/java/fr/zenity/desbugs/driver/Driver.java [hunk 2]
```diff
@@ -17,7 +17,7 @@
     WebDriver getWebDriver();
     void closeDriver();
 
-    default ChromeOptions chromeOptions(){
+    default ChromeOptions ChromeOptions(){
```
Comment (l-bachmann) [inline src/main/java/fr/zenity/desbugs/driver/Driver.java]: Why is it not camelCase ?

File: src/main/java/fr/zenity/desbugs/driver/Driver.java [hunk 3]
```diff
@@ -67,7 +67,7 @@ default OperaOptions OperaOptions(){
 
         operaOpts.setCapability(CapabilityType.ACCEPT_INSECURE_CERTS, true);
         operaOpts.setCapability(CapabilityType.ACCEPT_SSL_CERTS,true);
-        operaOpts.addExtensions(new File[]{(new File(Config.propConfig.extensionPath))});
+        //operaOpts.addExtensions(new File[]{(new File(Config.propConfig.extensionPath))});
```
Comment (l-bachmann) [inline src/main/java/fr/zenity/desbugs/driver/Driver.java]: Can you remove commented code ?

File: src/main/java/fr/zenity/desbugs/driver/LocalDriver.java [hunk 4]
```diff
@@ -1,23 +1,19 @@
...
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.openqa.selenium.opera.OperaDriver;
 
 public class LocalDriver implements Driver {
 
-    public final WebDriver webDriver = null;
+    private WebDriver webDriver = null;
 
     @Override
     public WebDriver getWebDriver() {
```
Comment (l-bachmann) [inline src/main/java/fr/zenity/desbugs/driver/LocalDriver.java]: If the driver is public, why use a getter function?

File: src/main/java/fr/zenity/desbugs/driver/LocalDriver.java [hunk 5]
```diff
@@ -26,4 +22,24 @@ public void closeDriver() {
...
+        );
+
+        return browser.toString().equals("firefox") ?
+               new FirefoxDriver(FirefoxOptions()) :
+               browser.toString().equals("edge") ?
+               new EdgeDriver(EdgeOptions()) :
+               browser.toString().equals("opera") ?
+               new OperaDriver(OperaOptions()) :
+               // default
+               new ChromeDriver(ChromeOptions());
```
Comment (l-bachmann) [inline src/main/java/fr/zenity/desbugs/driver/LocalDriver.java]: Maybe use a switch like the removed "getWebDriver" function. I think it is more readable.

File: configuration/configuration.properties.template [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+# default environment launched
+front.environment=
+api.environment=
+
+# default browser
+test.frontOffice=
+browser.default=
+headless=false
+extensionPath=
+device=
```
[the remark this answers is not in the document]
Comment (devGnode) [reply configuration/configuration.properties.template]: yes ^^

File: src/main/java/fr/zenity/desbugs/driver/Driver.java (hunk 2 above)
Comment (devGnode) [reply src/main/java/fr/zenity/desbugs/driver/Driver.java]: I dunno but you have right !

File: src/main/java/fr/zenity/desbugs/driver/LocalDriver.java (hunk 4 above)
Comment (devGnode) [reply src/main/java/fr/zenity/desbugs/driver/LocalDriver.java]: is not public, this var is private

File: src/main/java/fr/zenity/desbugs/Test.java (hunk 1 above)
Comment (devGnode) [reply src/main/java/fr/zenity/desbugs/Test.java]: yes but i will delete this class, from github not by this PR

**Pull request merged**